README.md

```diff
@@ -45,6 +45,53 @@ The following steps are needed to use the `System.Web` adapters with an ASP.NET
      ```csharp
      app.UseSystemWebAdapters();
      ```
+### Tests
+
+In most cases, there is no need to set up additional components for running tests, but if the subject under test makes use of `HttpRuntime`, you may need to start up the `SystemWebAdapters` service as outlined below:
```

Comment:
It's important here to call out that the tests need to be serialized. The example I gave in the issue was using XUnit test fixture to accomplish this. Otherwise, you'll get `InvalidOperationExceptions` when trying to start the host (if the runtime is set by a host, it can only be nulled out; it cannot be overriden to prevent this specific kind of scenario)

Reply:
Got it. I'm going to close this PR and open another PR to update AspNetCore.Docs. Thanks for the review!

Reply:
sounds good. once you open that, go ahead and link to this issue so we can track it.

Reply:
Just opened a PR in AspNetCore.Docs https://github.com/dotnet/AspNetCore.Docs/pull/30537. Closing this PR now.